middleware/dnssec/handler_test.go

```diff
@@ -129,6 +129,10 @@ func TestLookupDNSKEY(t *testing.T) {
 		}
 		resp := rec.Msg()
 
+		if resp.Authoritative != true {
```

Comment:
Some small nits:
You don't need the `!= true`, just do a `if !resp.Authoritative`